modules/zivid/camera.py

```diff
@@ -47,9 +47,127 @@ def __str__(self):
     def __eq__(self, other):
         return self.__impl == other._Camera__impl
 
+    def capture2d3d(self, settings):
```

Comment:
Should we change the "samples" directory and README snippets to use these new functions?

Reply:
Done